client/my-sites/promote-post-i2/components/post-item/style.scss

```diff
@@ -27,7 +27,10 @@ body.is-section-promote-post-i2 {
 		&-likes,
 		&-comments,
 		&-view,
-		&-promote {
+		&-promote,
+		&-sku,
+		&-price,
+		&-visitors {
```

Comment:
I don't think this class will be needed. In my last PR I updated the translations for the views to visitors, but it was just a label change. There is no need to match the label to the actual className. 

I think it will be better to just use the same class of the normal table views, because there is not change in how they both render.

THis same point can be applied to the other rule using `&-visitors` class.

Reply:
@sbarbosa  So you suggest to remove all this classes?

Reply:
Yes, I don't think we need it. But, only if you change the way we are importing the headers. 

From my point of view, both the normal and WOo version of the table can use the same header `views`. There is not need to have different header ids and class name just for that field that is a shared field.

Reply:
Fixed